hc/settings.py

```diff
@@ -20,7 +20,7 @@
 ALLOWED_HOSTS = [
     'healthchecks-akira.herokuapp.com',
 ]
-DEFAULT_FROM_EMAIL = 'healthchecks@example.org'
+DEFAULT_FROM_EMAIL = 'maryanne.nganga@andela.com'
```

Comment:
Please leave as is. This may be better for us as an environment variable. cc: @daisyndungu @Gathondu